Code/Commando/gamedata.cpp

```diff
@@ -123,7 +123,7 @@ WideStringClass			cGameData::WinText;
 //
 // hack
 //
-ULONG g_ip_override = INADDR_NONE;
```

Comment:
should all of these variables that use IP addresses be `uint32_t` instead?  It probably doesn't matter too much.